lib/components/RadioButton/RadioButton.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { render, screen, cleanup } from '@testing-library/react';
+import { afterEach, describe, expect, it, test, vi } from 'vitest';
+import { RadioButtonGroup } from './RadioButtonGroup';
+import { RadioButton } from './RadioButton';
+import '@testing-library/jest-dom/vitest';
+import React from 'react';
+
+afterEach(() => {
+  cleanup();
+});
+
+const Wrapper = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <RadioButtonGroup label="" value="" onChange={vi.fn()}>
```

Comment:
I think it is better to add some `label` and `value` than an empty string.

Reply:
I can add those. As this wrapper is not under the actual test I did not consider it crucial.